quickstart/101-attestation-provider/output.tf

```diff
@@ -0,0 +1,3 @@
+output "resource_group_name" {
```

Comment:
Can we rename this `output.tf` file to `outputs.tf` please due to the [recommended structure](https://www.terraform.io/language/modules/develop/structure)?